edelphi/src/main/resources/fi/metatavu/edelphi/liquibase/query-template-settings-enc.sql

```diff
@@ -0,0 +1,7 @@
+UPDATE LocalizedValue SET text = 'Ryhmä%201&Ryhmä%202' WHERE entry_id = (SELECT value_id FROM QueryPageTemplateSetting ts, QueryPageTemplateLocalizedSetting tls WHERE ts.key_id = (SELECT id FROM QueryPageSettingKey WHERE name = 'grouping.groups') and ts.id = tls.id) AND locale = 'fi';
+UPDATE LocalizedValue SET text = '-- Lisää otsikko --' WHERE entry_id = (SELECT value_id FROM QueryPageTemplateSetting ts, QueryPageTemplateLocalizedSetting tls WHERE ts.key_id = (SELECT id FROM QueryPageSettingKey WHERE name = 'time_serie.xAxisTitle') and ts.id = tls.id) AND locale = 'fi';
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal '-- Lisää otsikko --' 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=plsql%3AS1192)